caniusepython3/test/test_dependencies.py

```diff
@@ -69,6 +69,13 @@ def test_blocking_dependencies_no_project(self):
         got = dependencies.blocking_dependencies(['asdfsadfdsfsdffdfadf'], {})
         self.assertEqual(got, frozenset())
 
+    def test_blocking_dependencies_locators_fails(self):
+        py3 = {'py3_project': ''}
+        breaking_project = 'jekyll2nikola'
```

Comment:
Could you mock `distlib.locators.locate()` to throw an `AttributeError` instead? That way this test is dependent on jekyll2nikola continuing to trigger the error by not having a stable release. And please add a comment that this tests a work-around for https://bitbucket.org/pypa/distlib/issue/59/ .